DOI.js

```diff
@@ -93,25 +96,28 @@ function completeDOIs(doc) {
 	}
 }
 
-function retrieveDOIs(DOIs, doc) {
+function retrieveDOIs(DOIs, doc, provider) {
```

Comment:
Good idea. I will try to do this after :sleeping: . Please remind me if I forget about it.